### DISCUSSION TITLE
nu.get_account_feed_paginated() -  The request made failed with HTTP status code 401

### DISCUSSION THREAD

**Discussion by otavioz** [andreroggeri/pynubank#423] [Q&A]
Pessoal, a uns dias atrás notei que não estava recebendo as informações referentes a conta corrente NuBank, abaixo anexei o print do traceback. Tentei atualizar a biblioteca, utilizar a chamada antiga e dar _refresh_ no token, porém sem sucesso. Alguma luz? 

Obrigado

![image](https://github.com/andreroggeri/pynubank/assets/36630135/6b9e0127-1dd6-4741-b7f3-12f9244c5c07)

**Comment by andreroggeri** [comment]
Duplicado com a #419

**Reply by otavioz** [reply]
Obrigado, nao havia visto.
